sdk/core/azure-core/azure/core/_pipeline_client.py

```diff
@@ -98,6 +93,10 @@ def close(self):
         self.__exit__()
 
     def _build_pipeline(self, config, **kwargs): # pylint: disable=no-self-use
+        from .pipeline import Pipeline
+        from .pipeline.policies import (
```

Comment:
Let's even push that into after the `if policies is None`, we don't evem need any import if you provide your own policies